lib/sqlite.core.js

```diff
@@ -501,6 +501,16 @@ SQLitePluginTransaction.prototype.addStatement = function(sql, values, success,
       } else if (t === 'boolean') {
         //Convert true -> 1 / false -> 0
         params.push(~~v);
+      } else if (Array.isArray(v)) {
```

Comment:
perhaps you can used values.constructor === Array for consistency since it is already used on line 495 ?